esmvalcore/_recipe.py

```diff
@@ -576,6 +570,8 @@ def _read_attributes(filename):
 
 def _get_input_files(variable, config_user):
     """Get the input files for a single dataset (locally and via download)."""
+    if variable['project'] == 'CMIP5' and variable['frequency'] == 'fx':
```

Comment:
Note that this was moved here from https://github.com/ESMValGroup/ESMValCore/blob/a56e6110ac029557416f99be24ed66e17a2dd52c/esmvalcore/_data_finder.py#L494-L495. I think it's cleaner to set the correct facets when parsing the recipe instead of secretly changing this when looking for files.